templates/go/services/service.go.twig

```diff
@@ -95,7 +99,7 @@ func (srv *{{ service.name | caseUcfirst }}) {{ method.name | caseUcfirst }}({{
 	path := "{{ method.path }}"
 {% endif %}
 {{include('go/base/params.twig')}}
-{% if 'multipart/form-data' in method.consumes %}
+{% if 'multipart/form-data' in method.consumes and method.name | lower != "createexecution" %}
```

Comment:
I think we discussed to use method.type or so here

Reply:
This is to make sure it's not the `createexecution` method. the `method.type` won't cover all use-cases